NetUtilities/System/Reflection/Mapper.cs

```diff
@@ -43,6 +43,20 @@ public ReadOnlyList<FieldData> Fields
         public ReadOnlyList<MethodData> Methods
             => _methods ?? (_methods = _source.GetMethods(All).Select(x => new MethodData(x)).ToReadOnlyList());
 
+        /// <summary>
+        /// Contains data related to the type's methods - excluding members inhereted from the Object base class
+        /// </summary>
+        public ReadOnlyList<MethodData> MethodsExcludingObjectMembers
+            => _methods ?? (_methods = _source.GetMethods(All)
+            .Where(x => x.DeclaringType != typeof(Object)).Select(x => new MethodData(x)).ToReadOnlyList());
```

Comment:
If you're adding this, declare a field for it, because having 3 properties with 1 field will lead to unexpected results. For example, if you get MethodsExcludingObjectMembers, and then later you want to use MethodsDeclaringTypeOnly, it will return the former all the time because they're sharing the same field, which is set once.

Reply:
Hi JustNrik, 
Thanks for the feedback I'll sort it out.